style.css

```diff
@@ -0,0 +1,96 @@
+/* initial styling */
+body,
+ul {
+  margin: 0;
+  padding: 0;
+  background-color: #191765;
+  color: white;
+}
+
+li {
+  list-style: none;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+a {
+  color: white;
+}
+
+/* common classes */
+
+/* Main */
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 24px;
+}
+
+h3 {
+  font-family: Arial, sans-serif, 'Roboto';
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 16px;
+}
+
+h1 {
+  color: #7d73fc;
+  text-align: center;
+  font-weight: 700;
+  font-size: 48px;
+  line-height: 72px;
+}
+
+h2 {
+  text-align: center;
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 44px;
+  margin-bottom: 40px;
+}
+
+p {
+  margin: 0 24px;
+  text-align: center;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  margin-bottom: 40px;
+}
```

Comment:
- [ ] You seem to have missed some Figma design guidelines in this your headline section when it comes to typography. You should add the correct `font-family` and `color` to your `<h1> <h2> and <p>` tags. You can check the required values  by clicking on the specific element in the Figma template and copying the design parameters needed as can be seen below


![image](https://user-images.githubusercontent.com/90556221/164560378-7ec389fd-f1f7-432d-88be-3e16c8874e5e.png)